c/LinkedList.c

```diff
@@ -0,0 +1,99 @@
+// C program to Create and Sort a Single Linked List
+
+#include<conio.h>
+#include<stdio.h>
+#include<stdlib.h>
+typedef struct node
+{
+     int info;
+     struct node *next;
+}nodetype;
+nodetype* insert(nodetype*);
+void sort(nodetype*);
+void main()
+{
+     nodetype* left=NULL;
+     nodetype* right=NULL;
+     int ch,c;
+     printf("Enter 1 for insert \nEnter 2 for sort \nEnter 3 for exit");
+     do
+     {
+          printf("\nEnter choice: ");
+          scanf("%d",&ch);
+          switch(ch)
+          {
+               case 1:
+               right=insert(right);
+               if(left==NULL)
+               {
+                   left=right;
+               }
+               break;
+               case 2:
+               sort(left);
```

Comment:
@omprakashchhetri after sorting, the menu-driven is getting terminatted, try to continue the program back to menu-driven.

Reply:
possibly add a `break` so that it does not fall to `case 3` and exits.

Reply:
Sure !